Update IO rename form, refs# 13530

sbreker opened the pull request:
- Removed modal and replaced with alert class
- Updated to use b5 checkbox classes
- Updated layout using b5 columns

sbreker committed:
Update IO rename form, refs# 13530

- Removed modal and replaced with alert class
- Updated to use b5 checkbox classes
- Updated layout using b5 columns

jraddaoui commented on plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php:
```diff
@@ -47,36 +30,64 @@
+              <div class="col-md-3">
+                <div class="rename-form-field-toggle form-check">
+                  <input class="form-check-input" type="checkbox" id="rename_enable_title" checked>
+                  <label class="form-check-label" for="flexCheckChecked">
```
`for="rename_enable_title"`

sbreker replied:
🤦

sbreker replied:
Fixed

jraddaoui commented on plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php:
```diff
@@ -47,36 +30,64 @@
+              <div class="col-md-3">
+                <div class="rename-form-field-toggle form-check">
+                  <input class="form-check-input" type="checkbox" id="rename_enable_slug" checked>
+                  <label class="form-check-label" for="flexCheckChecked">
```
`for="rename_enable_slug"`

sbreker replied:
Fixed

jraddaoui commented on plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php:
```diff
@@ -47,36 +30,64 @@
+            </div>
+            <hr />
+
+            <div id="rename-slug-warning" class="alert alert-danger" style="display: none;">
```
Add `role="alert"`.

sbreker replied:
Done

jraddaoui commented on plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php:
```diff
@@ -47,36 +30,64 @@
+                <div class="col-md-3">
+                  <div class="rename-form-field-toggle form-check">
+                    <input class="form-check-input" type="checkbox" id="rename_enable_filename" checked>
+                    <label class="form-check-label" for="flexCheckChecked">
```
`for="rename_enable_filename"`

sbreker replied:
Fixed

jraddaoui commented on plugins/arDominionB5Plugin/scss/_layout.scss:
```diff
@@ -34,7 +34,7 @@ body {
 .dropdown-menu {
   box-shadow: $box-shadow;
-  
+
```
Ups ... thanks!

jraddaoui replied:
Added this in a different commit, please rebase ;)

sbreker replied:
👍

jraddaoui reviewed (changes requested):
Looking great @sbreker!

With the dividers you added, I'd prefer to have each input on a single row, giving more space to fields like these ones that could be long, and avoiding all the whitespace under the check-boxes.

Added a few minor but important comments below.

sbreker committed:
Address feedback items

- added role='alert'
- for 'for' element correctly to associate the label with the input
field.

sbreker committed:
Updated layout of rename template

Updated the form layout so all fields are left justified and stacked.
<hr /> tags used to provide visual separation of the three sections.

sbreker commented:
Hi @jraddaoui - thanks for the feedback! The layout has been updated to remove the columns bringing the fields all over to the left side to give the long text input fields more room.

jraddaoui reviewed (approved):
Nice, thanks @sbreker!!
